tests/sources/test_notebooksource.py

```diff
@@ -256,25 +256,6 @@ def test_error_if_source_is_dir_suggest_scaffold(tmp_directory):
     assert 'ploomber scaffold' in str(excinfo.value)
 
 
-def test_error_if_parameters_cell_doesnt_exist():
-    with pytest.raises(SourceInitializationError) as excinfo:
-        NotebookSource(new_nb(fmt='ipynb', add_tag=False), ext_in='ipynb')
-
-    assert 'Notebook does not have a cell tagged "parameters"' in str(
-        excinfo.value)
-
-
-def test_error_missing_params_cell_shows_path_if_available(tmp_directory):
-    path = Path('nb.ipynb')
-    path.write_text(new_nb(fmt='ipynb', add_tag=False))
-
-    with pytest.raises(SourceInitializationError) as excinfo:
-        NotebookSource(path)
-
-    assert ('Notebook "nb.ipynb" does not have a cell tagged "parameters"'
-            in str(excinfo.value))
-
-
```

Comment:
Do we have tests to check that the parameter cell is added if the input notebook *does not* have it? I couldn't see them

you can use this as reference. read the source ipynb file with nbformat and check that the top cell has the parameters tag

Reply:
I think we already have `tests/sources/test_notebooksource.py::test_add_parameters_cell` in place. Would that be sufficient?

Reply:
Added in https://github.com/ploomber/ploomber/pull/971/commits/2c624eebce91c6953ee366a2ab0d730670efe14d